payments/utils.py

```diff
@@ -105,6 +110,51 @@ def _process_payment(order, payment, verbose=0):
         method=payment.get('payment_method', 'fiobank-transfer'),
     )
 
+    if settings.PAYMENT_PROCESS_EMAIL_NOTIFY:
+        event = order.event
+        template = EmailTemplate.objects.get(name='order_update_email')
+        subject = _('Your ticket for {event}.'.format(variable_symbol=order.variable_symbol,
```

Comment:
yes, my fault I was experimenting with different subjects and forgot to remove. At the end I decided to use the same subject so in most clients the email end in the same thread as registration email which make sence.